client/basic-authorship/src/basic_authorship.rs

```diff
@@ -57,6 +57,8 @@ pub const DEFAULT_BLOCK_SIZE_LIMIT: usize = 4 * 1024 * 1024 + 512;
 
 const DEFAULT_SOFT_DEADLINE_PERCENT: Percent = Percent::from_percent(50);
 
+const LOG_TARGET: &'static str = "sc-basic-authorship";
```

Comment:
```suggestion
const LOG_TARGET: &'static str = "basic-authorship";
```

If you don't specify a log target, the crate name (sc-basic-authorship) will be the default prefix.

Reply:
Oh, did not know.